website/source/docs/drivers/docker.html.md

```diff
@@ -26,7 +26,7 @@ The `docker` driver supports the following configuration in the job specificatio
 * `network_mode` - (Optional) The network mode to be used for the container.
    Valid options are `default`, `bridge`, `host` or `none`. If nothing is
    specified, the container will start in `bridge` mode. The `container`
-   network mode is not supported right now.
+   network mode is not supported right now, this case also defaults to `bridge`.
```

Comment:
My concern is the case where an user, being familiar with docker, tries to use `container` as a network mode. Nomad will launch the container in `bridge` mode. I wanted to clarify that.